server/src/main/java/org/opensearch/action/ActionModule.java

```diff
@@ -675,6 +681,10 @@ public <Request extends ActionRequest, Response extends ActionResponse> void reg
         // Remote Store
         actions.register(RestoreRemoteStoreAction.INSTANCE, TransportRestoreRemoteStoreAction.class);
 
+        // Decommission actions
+        actions.register(PutDecommissionAction.INSTANCE, TransportPutDecommissionAction.class);
+        actions.register(GetDecommissionAction.INSTANCE, TransportGetDecommissionAction.class);
```

Comment:
This could be `GetDecommissionStatusAction`

Reply:
Apart from status, it also gives what attribute is actually decommissioned. LMK if you think `GetDecommissionAction` could be misleading?

Reply:
Named it GetDecommissionStateAction